### DISCUSSION TITLE
Option to passed  different 'sgptrc' files

### DISCUSSION THREAD

**Discussion by rcspam** [TheR1D/shell_gpt#618] [Ideas]
Hi,
Thanks for your incredible job.

Is it possible to add an option to passed differents 'sgptrc' files.
This would allow you to create personalized sgptrc files and choose according to your needs.
For example, I have a config file for 'openai' and one for 'groq' for example without having to edit it each time.
`$ sgpt --config path_to_your_sgptrc ....`

It coulb be very cool.
rcspam

**Comment by mariano-soto** [comment]
I think this is a great idea. I'm going to check this out.
